src/main/java/racingcar/Cars.java

```diff
@@ -0,0 +1,50 @@
+package racingcar;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Cars extends Condition {
+	private List<Car> cars = new ArrayList<>();
+
+	public Cars(String[] cars) {
+		super(new NameCondition());
+		for (String carName : cars) {
+			validName(carName);
+		}
+	}
+
+	private void validName(String carName) {
+		if (validInput(carName)) {
+			this.cars.add(new Car(carName));
+		}
+	}
+
+	public List<Car> getCars() {
+		return cars;
+	}
+
+	public List<Car> findWinner() {
+		int max = 0;
+		List<Car> winners = new ArrayList<>();
+		for (int i = 0; i < cars.size(); i++) {
+			max = bestPosition(max, i);
```

Comment:
JDK API를 활용해 보시는 것도 추천드립니다 😃 
```suggestion
			max = Math.max(max, cars.get(i).getStatus());
```